### DISCUSSION TITLE
Compressor Reduction in noise reduction mode for Vaillant heat pumps

### DISCUSSION THREAD

**Discussion by BenKrae** [john30/ebusd#1148] [Q&A]
Hello everyone,

has anyone figured out the parameter for the compressor reduction during noise reduction mode for vaillant heat pumps?
Its possible to change this parameter at the hmu between 40%-60%.
I'm hoping this is a writeable parameter on the ebus because i want to use to adjust the consumed electric power to photovoltaic power and time varying electricy prices.

**Comment by apreber** [comment]
Hi,
I am interested in exactly the same parameter. I modified the parameter 3 times between 2 values and looked into the ebusd.log. It seems not to be sent on the bus as this probably only was an internal change in the hmu.
But I am pretty sure that this parameter can be accessed via ebus as vaillant seems to have the possibility to at least look at all parameters with a special windows software or with the module that connects the heat pump to the internet.

**Comment by Redskyer** [comment]
Hi,
I'm also looking for that parameter. Are there any updates?
As far as I can see, I can read "NoiseReduction" and "NoiseReductionFactor" and I can write "NoiseReductionFactor", but without the possibility of activating "NoiseReduction", there is no need for the configuration of "NoiseReductionFactor".

**Comment by ectoplasmosis** [comment]
I would also like to be able to set the Noise Reduction mode, as well as set the noise reduction value. Following to see if this will become possible.
